### DISCUSSION TITLE
How teleport player to another server from Fabric mod code

### DISCUSSION THREAD

**Discussion by AmibeSkyfy16** [JustPyrrha/QForward#5] [Q&A] [answered]
Hi, i have some diffictulties to teleport player between servers.
I really don't understand how it works

this is my code: https://postimg.cc/FdG30xZC

**Comment by AmibeSkyfy16** [comment]
Also I could run the command "/server map2" for a given player from code, but I can't find anything about how to code this with Fabric

**Reply by AmibeSkyfy16** [reply]
I have this now, but still error: server.getCommandManager().execute(serverPlayerEntity.getCommandSource(), "server map2");
but server.getCommandManager().execute(serverPlayerEntity.getCommandSource(), "deop name"); work

**Comment by JustPyrrha** [comment, marked as answer]
fabric-forwarding doesn't handle anything regarding sending players between servers, it just accepts the login format from proxy servers. You should ask in the support area for whatever proxy you're using for your network.

**Reply by AmibeSkyfy16** [reply]
Ok thanks for your answer. i will investigate
